content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -210,6 +210,10 @@ The following configuration variables are available **only** when using automati
 Enables or disables all automatic instrumentation. Setting the environment variable to `false` completely disables the CLR profiler. For other configuration methods, the CLR profiler is still loaded, but traces will not be generated. Valid values are: `true` or `false`.<br>
 **Default**: `true`
 
+`DD_DBM_PROPAGATION_MODE`
+: Enables DBM to APM link when set to `'service'` or `'full'`. The `'service'` option enables the connection between DBM and APM services. The `'full'` option enables connection between database spans with database query events. Available for Postgres and MySql.<br>
```

Comment:
Can we phrase this in a way that allows us to say `Database Monitoring` and link to the DBM docs in general? Something like `Enables linking between data sent from APM and the Database Monitoring product....` if that's not too wordy, with a link to https://docs.datadoghq.com/database_monitoring/

Reply:
(here & in the framework page as well)

Reply:
One existing feature we have that links APM and Logs is a feature we tend to call "Log Correlation" (see the docs [here](https://docs.datadoghq.com/tracing/other_telemetry/connect_logs_and_traces/dotnet/?tab=serilog#configure-injection-in-logs)). Would some of that language be suitable here?

Reply:
oh wow, in general this is language we should steal for the main documentation page. I'll work on that separately. 

For this chunk, I don't think we need big changes to your language - just a way to insert the full phrase `Database Monitoring` with a link before we refer to it using the initials DBM. I think if we just changed the first sentence to `Enables linking between data sent from APM and the Database Monitoring product when set to 'service' or 'full'` that would do it.

Reply:
> I think if we just changed the first sentence to Enables linking between data sent from APM and the Database Monitoring product when set to 'service' or 'full' that would do it.

Sounds good. I'll go ahead with this suggestion 👍🏼

Reply:
Done in 69bfe6ece90c7bb51c1f11e37cc6e4d846f98d47